scripts/ci-build-macos-package.sh

```diff
@@ -1,12 +1,10 @@
 #!/bin/bash
-pushd test/desktop
-  ./features/scripts/build_maze_runner.sh
-  if [[ $? != 0 ]]; then
-    popd
-    exit $1
-  fi
-
-  pushd features/fixtures/maze_runner
-    zip -r Mazerunner-$UNITY_VERSION.app.zip Mazerunner.app
+./features/scripts/build_maze_runner.sh
+if [[ $? != 0 ]]; then
   popd
```

Comment:
Is this still required?

Reply:
You're right, it's a copy and paste error.  I'll remove it on the branch I'm now working on.